templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% block content %}
+
+<div class="container">
+
+    <div class="row">
+        <div class="col-md-12">
+            <form action="" method="post">{% csrf_token %}
+                {% if form.email.errors %} {{ form.email.errors }} {% endif %}
+                    <p>email address: {{ form.email }}</p> 
```

Comment:
same here. I don't think there are any extra fields, but `{{ form.as_p }}` will handle the errors, label, and field

Reply:
adding changing email stuff to "{{ form.as_p }}" to our "after the election" list